projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -0,0 +1,113 @@
+import { Gridicon } from '@automattic/jetpack-components';
+import { Popover } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from 'react';
+import { PRODUCT_STATUSES } from '../../../constants';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import { timeSince } from '../../../utils/time-since';
+
+import './style.scss';
+
+const ProtectValueSection: React.FunctionComponent = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { status } = detail;
+	const isProtectActive =
+		status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
+
+	return isProtectActive ? <WithProtectValueSection /> : <NoProtectValueSection />;
+};
+
+export default ProtectValueSection;
+
+const WithProtectValueSection: React.FunctionComponent = () => {
+	const { protectStatus } = getMyJetpackWindowInitialState();
+	const lastScanTime = protectStatus?.last_checked;
+	const timeSinceLastScan = lastScanTime ? timeSince( Date.parse( lastScanTime ) ) : '...';
+
+	const lastScanText = sprintf(
+		/* translators: %s is how long ago since the last scan took place, i.e.- "17 hours ago" */
+		__( 'Last scan: %s', 'jetpack-my-jetpack' ),
+		timeSinceLastScan
+	);
+	return <ValueSection isProtectActive={ true } lastScanText={ lastScanText } />;
+};
+
+const NoProtectValueSection: React.FunctionComponent = () => {
+	const { plugins, themes } = getMyJetpackWindowInitialState();
+	const pluginsCount = Object.keys( plugins ).length;
+	const themesCount = Object.keys( themes ).length;
+
+	const pluginsThemesText = sprintf(
+		/* translators: %1$d is the number (integer) of plugins and %2$d is the number (integer) of themes the site has. */
+		__( '%1$s plugins & %2$s themes', 'jetpack-my-jetpack' ),
+		pluginsCount,
+		themesCount
+	);
+
+	return <ValueSection isProtectActive={ false } lastScanText={ pluginsThemesText } />;
+};
+
+const ValueSection: React.FunctionComponent< {
+	isProtectActive: boolean;
+	lastScanText: string;
+} > = ( { isProtectActive, lastScanText } ) => {
+	const isMobileViewport: boolean = useViewportMatch( 'medium', '<' );
+	const [ isVisiblePopover_PluginsThemes, setIsVisiblePopover_PluginsThemes ] = useState( false );
```

Comment:
Mixing snake case and camel case is not a standard practice that I've seen personally 🤔 

Since these variables are restricted to this file, I think it would be okay to just name them `isPopoverVisible` and `setIsPopoverVisible`

Reply:
Same with `togglePopover_PluginsThemes`

Reply:
Yeah, hmm.. I only did this because there is going to be three(3) different info popovers on the card.  1 for PluginsThemes, 1 for ScansThreats, and 1 for AutoFirewall, each of which will need their own 'visible' state.

Right now for this PR, while there is only 1 popover, I'll change this over to `isPopoverVisible` and `setIsPopoverVisible` (and `togglePopover()`) for now.
But then in following PR's when I add the additional popovers I'll need to come up with a better way to keep track of the state of all of the popovers.  Ideally, only one popover should be able to be opened at a time, meaning that when the user opens a popover on the card, any other popovers that may be previously opened, should close first before opening the currently selected popover, if that makes sense.. Anyway I'll deal with that in upcoming PR(s).

Reply:
Oh in that case it's okay to specify which popover it is, but we should probably stick to just camel case when naming it 😄